### ISSUE TITLE
Cannot turn OFF flashlight on UWP tablet

### ISSUE THREAD

**Issue by tsoen** [PTCInc/vuforia-engine#90]
**Summarize your feature feedback**
Hello, 
Yet another question regarding flashlight, but I think the issue might be different than #89 so I created a new ticket.
On a Unity-UWP build, I am able to turn ON the flashlight of the device.
However, once it is done, I can never turn it OFF. There is no error message at all.

Device : Dell Latitude 7230 Rugged Extreme Tablet. Windows 11. Windows SDK version 10.0.22621.

Here is my code :
	
```
public void EnableFlashLight(bool enable)
{
    Logger.Error($"EnableFlashLight {enable}");
	
    // Try to enable/disable flash. If it did not work, assume the flash state did not change.
    if (_vuforiaBehaviour.CameraDevice.SetFlash(enable))
    {
        _isFlashActivated = enable;
    
        Logger.Error($"EnableFlashLight _isFlashActivated = {_isFlashActivated}");
    }
}
```

My logs confirm that we always enter the `if` condition whether the `enable` variable was true or false. No error is thrown, but the flashlight never shuts down.

**Business impact**
Flashlight feature cannot be used.

**Workaround**
Did not think of a workaround yet.

**Comment by ptc-pscheper** [comment]
@tsoen Definitely another issue! 😉 Thank you for the bug report. I will reach out to the R&D team as I do not have a UWP device available at the moment to reproduce the issue. Have you tried to reproduce this on another UWP device?

**Comment by tsoen** [comment]
Thank you for the answer @ptc-pscheper, glad there may be some hope to solve this one !

I think we have a Surface Pro 6 at the office, though I am not sure it has a flashlight.
I'll definitely check once I get back to the office this week.

**Comment by tsoen** [comment]
Hello @ptc-pscheper, unfortunately the only other UWP device we have is a Surface Pro 6 and I just confirmed it does not have a flashlight, so I am not able to reproduce the issue that appeared on the Dell Latitude 7230 Rugged Extreme Tablet 😕

**Comment by ptc-pscheper** [comment]
Hi @tsoen, sadly the team will be unable to go through this bug report because of the following reasons:

- The Dell Latitude 7230 is not a recommended device. 
- The little sister, Dell Latitude 7220, has been consistently a very unpopular device and will drop from our recommended device list in upcoming releases.

Overall this means I will need to close this bug report. I am sorry I am not able to be of more help...

**Comment by tsoen** [comment]
For the record, I can confirm the issue was not related to this specific tablet model.
It was fixed in Vuforia Engine v10.22.5 : "Fixed an issue that prevented disabling torch mode on UWP devices." (https://developer.vuforia.com/library/release-notes/vuforia-engine-release-notes#vuforia-engine-v10-22-5)
